### DISCUSSION TITLE
Variation in partition coefficient methods based on dose - which to choose?

### DISCUSSION THREAD

**Discussion by Cheruvh** [Open-Systems-Pharmacology/Forum#1553] [Questions & Problems]
Hi,

I come across a situation while performing Parameter identification (PI) in PK-Sim using the 5 partition methods. Now, at a 400 mg dose, the drug PK profile is well-predicted with the 'PK-SIM standard' method as the partition coefficient method whereas it is poorly predicted for a supra-therapeutic dose of 800 mg. However, Poulin and Theil's method predicted the PK profile of 800 mg very well as shown below. I tried to do PI together with 400 mg and 800 mg PK profiles and got the PK-Sim method as best but not really explain the distribution of 800 mg. Here the drug shows a non-linear PK behavior across the doses 200, 400, and 800 mg. 

800 mg PI simulation:
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/103236850/4c3848d3-06df-4b51-939b-df8935dec595)
Red dots-observed data
Green line-Poulin and Theil PC method simulation
Blue line-PK-Sim method simulation

Please advise the best method or option to get a generalized model. For a generalized model, I tried to use binding partners like lysosomes and protein partners to fix the distribution phase. However, the Tmax got shifted to half of its original like 0.75 hr from the actual 1.5 to 2  hr. I have no idea whether it is the right way to do it or not. Moreover, I observed when using the binding partners, and optimizing the tissue partitioning values by fixing them based on preclinical data, I felt the 5 PC methods optimization is not working anymore and the model lost its integrity as there is a manual optimization into play. Do I have to try anything more here? 

Thanks in advance.

**Comment by StephanSchaller** [comment]
Certainly, if you fix partitioning values, the PC method optimization wont work any more and you either fit individual uncertain PC values directly together with the processes you added and thenusual fitting parameters (clearance, LogP, fu...).

**Comment by Cheruvh** [comment]
Hi Stephan,
Thank you. 

Apart from the fixed partitioning values, I think the protein binding partners could also alter the normal control over PC method optimization in PKSIM, is it right?

I am curious to know the scientific solution for a generalized mechanistic PBPK model. So, could you suggest anything to make the model work at both 400 and 800 mg? Do I have to introduce any non-linearity parameter in the distribution phase for this, as elimination is approximately close between the PKSIM and Poulin and Theil methods for 800 mg? I read the literature where logP(oil/water) is introduced in Poulin and Theil and Berezhkovsky methods which is missing in the PK-Sim PC method. Please advise.

**Comment by tobiasK2001** [comment]
Hi,

sorry for jumping in late but it looks like you are speaking about 400 and 800 mg p.o. doses, right? So how does it look like if you overlay your dose normalized observed data? Could there be some difference on the absorption side between 400 and 800 mg. Decisions about the distribution models can be done best based on i.v. data.

Best, Tobias

**Reply by Cheruvh** [reply]
Hi Tobias,
Thank you.

They are orally dosed profiles and we don't have the IV profiles. Absorption is nearly similar and it is very rapid reaching the Cmax in 1 hr and can see only one sample point before Cmax. The dose normalised plots proved that they are non-linear. This tricky compound is a Pgp, cyp2D6 and cyp3A substrate and also a moderate to mild inhibitor to Pgp, Cyp2D6 and TDI on Cyp3A. This a basic amine compound assumed a chance of lysosomal trapping. But still lacks a model based prediction for its distribution behavior.

**Reply by tobiasK2001** [reply]
Yes that really sounds tricky. It is likely that you have flip flop kinetics. And of course same other fancy stuff like lysosomal trapping can go on, too. Do you have some low dose data, which is dose linear? I am not sure, if you can find a consistent distribution model without iv data.

Best, Tobias

**Comment by Cheruvh** [comment]
Hi Folks,
I guess the lysosomal trapping consideration in PK-Sim through a protein binding partner is like linear kinetics following a first-order process with kd and koff kinetics. However, there is a research paper (https://dmd.aspetjournals.org/content/dmd/47/1/49.full.pdf) stating that lysosomal trapping might be non-linear and dose-dependent with saturation kinetics. In my case, I can see 200 and 400 mg PK data is well explained by PK-Sim standard PC method without any need for lysosomal trapping. However, for higher doses like 800 mg and above, lysosomal trapping is profoundly having an effect, and incorporation of it showed the best simulation during model development. Now my question is - is it possible to consider lysosomal trapping in Michaelis-Menten kinetic fashion (non-linear kinetics) the same as a CYP metabolism reaction in PK-Sim (assuming lysosomal trapping as a metabolizing enzyme rather than a protein binding partner)?  Please advise, correct me if I am wrong here, and thanks in advance.
